services/movie_session.py

```diff
@@ -13,7 +13,7 @@ def create_movie_session(
     )
 
 
-def get_movies_sessions(session_date: str = None) -> QuerySet:
+def get_movies_sessions(session_date: str = None) -> QuerySet[MovieSession]:
     queryset = MovieSession.objects.all()
     if session_date:
         queryset = queryset.filter(show_time__date=session_date)
```

Comment:
Consider using a `datetime.date` object for the `session_date` parameter in the `get_movies_sessions` function to ensure type safety and consistency when filtering by date.